packages/salesforcedx-vscode-expanded/package.json

```diff
@@ -43,6 +43,7 @@
     "dbaeumer.vscode-eslint",
     "esbenp.prettier-vscode",
     "chuckjonas.apex-pmd",
+    "pmd.apex-pmd",
```

Comment:
@mingxuanzhangsfdx is the assumption here that an "unknown" extension will fail silently regardless of registry

Reply:
@mingxuanzhangsfdx nit: is it worth adding a comment here why there are duplicate entries for future readers?